build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -585,8 +586,102 @@ export const handlers: Handler[] = [
             return { resolved: resolved };
         },
     },
+    {
+        name: "npm-package-json-prettier",
+        match,
+        handler: (file) => {
+            let json;
+
+            try {
+                json = JSON.parse(readFile(file));
+            } catch (err) {
+                return "Error parsing JSON file: " + file;
+            }
+
+            const hasPrettierScript = Object.prototype.hasOwnProperty.call(
+                json.scripts,
+                "prettier",
+            );
+            const hasPrettierFixScript = Object.prototype.hasOwnProperty.call(
+                json.scripts,
+                "prettier:fix",
+            );
+            const hasFormatScript = Object.prototype.hasOwnProperty.call(json.scripts, "format");
+
+            if (!(hasPrettierScript && hasPrettierFixScript && hasFormatScript)) {
```

Comment:
I think this would be more readable without the outer `if`, and after the three individual checks something like `return missingScripts.length > 0 ? <the error> : undefined;`

Reply:
Removed the outermost if-condition and replaced with ternary operator.